repos/system_upgrade/el7toel8/actors/unsupportedupgradecheck/actor.py

```diff
@@ -0,0 +1,66 @@
+import os
+import json
```

Comment:
Wrong import order

Reply:
os isn't even used, we should check the linter if it works like it should